### DISCUSSION TITLE
Lodestar Standup 2024-43 (December 3, 2024)

### DISCUSSION THREAD

**Discussion by philknows** [ChainSafe/lodestar#7263] [Standups]
### Meeting Info
[December 3, 2024, 15:00-16:00 UTC](https://savvytime.com/converter/utc-to-pst-est-cet-ist-ict/dec-03-2024/3pm)
Zoom: shared on #lodestar-general ChainSafe Discord channel at 15:00 UTC

### Agenda
- Open Discussion
  - v1.24 Planning
- Team Updates
- Closing Remarks

### Instructions
Please post here any topics to be added for Standup discussions. Examples of agenda items are issues, discussions, proposals which require feedback, consensus from the team and/or community members.

**Comment by twoeths** [comment]
I want to revisit our mission "The production-grade JavaScript platform that accelerates Ethereum consensus R&D and maximizes its robustness"

what can we do to support that mission? one idea could be:
- each member should join at least an EIP/devnet
- for big EIPs/devnets, we need multiple members to work on them

I'm not sure if it's dynamic enough, the goal is to have more people to work on different EIPs/devnets and have lodestar as one of the first clients to join different EIPs/devnets

**Comment by nflaig** [comment]
we could quickly discuss about the giraffe banner https://github.com/ChainSafe/lodestar/pull/7233 if there is time left, not sure we want this for devnets already as the giraffe banner is quite huge as it is right now

**Reply by philknows** [reply]
Sounds like a good idea. We will also do a quick PR grooming to clean out and figure out what can potentially make it to v1.24 by end of week.

**Comment by philknows** [comment]
Closing for #7289
